src/topic.ts

```diff
@@ -274,13 +302,19 @@ export class Topic {
    *   const exists = data[0];
    * });
    */
-  exists(callback) {
-    this.getMetadata(err => {
+  exists(callback: ExistsCallback) {
+    this.getMetadata((err) => {
       if (!err) {
         callback(null, true);
         return;
       }
-      if (err.code === 5) {
+      let code = 0;
```

Comment:
Can we clean this up to not entirely change the code?  casting as any is ok

Reply:
Done!